EasyPost/Services/CarrierAccountService.cs

```diff
@@ -68,6 +68,12 @@ public async Task<CarrierAccount> Create(Parameters.CarrierAccount.Create parame
                 throw new MissingParameterError(nameof(parameters.Type));
             }
 
+            // stop the user from using a generic Create parameter set when creating a carrier account with a custom workflow (e.g. FedExAccount)
+            if (parameters.GetType() == typeof(Parameters.CarrierAccount.Create) && Constants.CarrierAccountTypes.CarrierTypesWithCustomWorkflows.Contains(parameters.Type))
```

Comment:
thought: one thing we should consider is that CanadaPost is also a carrier that needs a custom workflow (same endpoint as old) - did we want to handle that in this PR too?

Reply:
So it needs a custom parameter set, but needs to hit the generic endpoint, or both?

Reply:
Disregard this actually, the endpoint and param key is the same - it's "custom" because you have to pass a special param to tell it not to authenticate via the browser but users can still use the function to do that without us needing to change anything.